websites/B/BreathFM/presence.ts

```diff
@@ -0,0 +1,59 @@
+const presence = new Presence({
+		clientId: "846282107462352927",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	let presenceData: PresenceData;
+	if ("mediaSession" in navigator && navigator.mediaSession.metadata !== null) {
+		presenceData = {
```

Comment:
Also why not just do this?
```suggestion
	let presenceData: PresenceData;
	if ("mediaSession" in navigator && navigator.mediaSession.metadata !== null) {
		const presenceData: presenceData = {
```